client/state/selectors/get-parent-comment.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getPostCommentsTree } from 'state/comments/selectors';
+
+/**
+ * Returns the parent comment of a given comment
+ *
+ * @param {Object} state Redux state
+ * @param {Number} siteId Site identifier
+ * @param {Number} postId Post identifier
+ * @param {Number} commentId Comment identifier
+ * @return {Object} The parent comment
+ */
+export const getParentComment = ( state, siteId, postId, commentId ) => {
+	const commentsTree = getPostCommentsTree( state, siteId, postId, 'all' );
+	const parentCommentId = get( commentsTree, [ commentId, 'data', 'parent', 'ID' ], 0 );
+
+	return parentCommentId ? get( commentsTree, [ parentCommentId, 'data' ], {} ) : {};
```

Comment:
no need for the ternary check because you are using defaults on `get`

```js
return get( commentsTree, [ parentCommentId, 'data' ], {} );

Reply:
Oh gosh, good catch, not sure how I missed that... it was literally 1 character to the left!